FJAM-Studios/pixtery
# Cloud storage

You can now download a puzzle using a URL if you know the public key.  If you have already downloaded the puzzle it will not download again, but it might be nice if it automatically redirected you to that puzzle.  The downloaded puzzle just gets added to state using setReceivedPuzzles so it is not permanently stored.

To test:
- expo start as normal
- enter this url into your mobile browser: exp://192.168.1.12:19000?puzzle=test123

Theoretically, the url should change automatically depending on the environment the app is running in (eg. development in expo, deployed in expo, or a standalone app) but there's no way to test that until we move out of development

The url is generated in home.tsx in the function shareLink. Currently it just logs the url but that can be expanded  to create the SMS
